dockerfile

```diff
@@ -0,0 +1,13 @@
+
+#Using python
+FROM python:3.11.7
```

Comment:
The production posit connect server where we deploy to currently runs 3.8.18, so we might want to downgrade this to be in sync with the server's python version to avoid any compatibility issues.